api/pkg/s3/bucketlifecycleput.go

```diff
@@ -214,7 +272,7 @@ func (s *APIService) BucketLifecyclePut(request *restful.Request, response *rest
 
 	// Create bucket with bucket name will check if the bucket exists or not, if it exists
 	// it will internally call UpdateBucket function
-	res, err := s.s3Client.UpdateBucket(ctx, bucket)
```

Comment:
imho, this is incorrect. why are we tampering with the whole bucket here, just to set some new lifecycle rules ?